app/code/Magento/Catalog/Model/Product/Type.php

```diff
@@ -164,7 +161,7 @@ public function priceFactory($productType)
      * Get Product Price Info object
      *
      * @param Product $saleableItem
-     * @return \Magento\Framework\Pricing\PriceInfoInterface
```

Comment:
This string should be reverted.
Theoretically, this class can be used in WebAPI requests.

The current implementation of WebAPI "request/response" parser has a limitation:

> Valid object types include a fully qualified class name or a fully qualified interface name.
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/service-contracts/service-to-web-service.html

Reply:
@hws47a Please make changes according to @naydav comment.